web/themes/custom/novel/templates/fields/field--media--field-media-file.html.twig

```diff
@@ -0,0 +1,9 @@
+{% for item in items %}
+   <a href="{{ item.content['#file']|file_url }}" class="link-with-icon arrow__hover--right-small">
+    <div class="link-with-icon__icon ">
+      <img class="invert" src="/themes/custom/novel/assets/dpl-design-system/icons/collection/Ebook.svg" alt="" />
+    </div>
+    {{ item.content["#description"] }}
+    {{ source("/themes/custom/novel/assets/dpl-design-system/icons/arrow-ui/icon-arrow-ui-small-right.svg") }}
```

Comment:
Please look into moving the icon path into a variable.

[We have already done this for paragraphs](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/f592784bf837dbee012bbde2293b2a78676ffbb9/web/modules/custom/dpl_paragraphs/dpl_paragraphs.module#L18-L18) and [used it in a template](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/f592784bf837dbee012bbde2293b2a78676ffbb9/web/themes/custom/novel/templates/paragraphs/paragraph--links.html.twig#L5-L5), but I think [it should be possible to do this for all templates in the theme](https://drupal.stackexchange.com/a/237666).